PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m

```diff
@@ -42,7 +43,13 @@ - (PNLiteAdRequestModel *)createAdRequestWithZoneID:(NSString *)zoneID
     adRequestModel.requestParameters[HyBidRequestParameter.coppa] = [HyBidSettings sharedInstance].coppa ? @"1" : @"0";
     [self setIDFA:adRequestModel];
     adRequestModel.requestParameters[HyBidRequestParameter.locale] = [HyBidSettings sharedInstance].locale;
-    if (![HyBidSettings sharedInstance].coppa) {
+    
+    NSString* privacyString =  [[HyBidUserDataManager sharedInstance] getIABUSPrivacyString];
+    if (![[HyBidUserDataManager sharedInstance] isCCPAOptOut] && !([privacyString length] == 0)) {
```

Comment:
We should fix this condition. There's also an error on Android. We should send the consent string if it's set even if the user opts out. So I'd just keep the check to making sure its not nil or empty.

Reply:
@eros902002 `isCCPAOptOut` is removed in the latest commit. Please check :D